docs/component.md

```diff
@@ -0,0 +1,60 @@
+@function react-view-models.Component Component
+@parent react-view-models 1
+
+@description Connects a [DefineMap](./can-define/map/map.html) class to a React component to create an auto-rendering component with an observable view-model.
```

Comment:
You can avoid linking directly to html files by using syntax like `[can-define/map/map]` or `[can-define/map/map DefineMap]`.